### PR TITLE
Use correct environment variable for admin email when installing

### PR DESCRIPTION
**Pull request by kehh**

### CODE REVIEW

File: bin/installOrUpdate [hunk 1]
```diff
@@ -37,7 +37,7 @@ function runProcess(Process $p): void
     });
 }
 $profile = getenv("PROFILE");
-$email = getenv("ADMIN_EMAIL");
+$email = getenv("CA_ADMIN_EMAIL");
```
Comment (bri-gaia) [inline bin/installOrUpdate]: Does this conflict with another project? should we maybe check for both?

**Comment by kehh** [comment]
@bri-gaia was there something you wanted changed in this?

**Comment by bri-gaia** [comment]
> @bri-gaia was there something you wanted changed in this?

I don't know how to use git hub, sorry I have a question in a pending review?

"Does this conflict with another project? should we maybe check for both?"

File: bin/installOrUpdate (hunk 1 above)
Comment (bri-gaia) [reply bin/installOrUpdate]: @kehh  I found the button >.< sorry, no wonder you hadn't answered me :P

**Comment by kehh** [comment]
OK have fixed that.

**Pull request merged**